en/drafts/originals/space-place-gazetteers.md

```diff
@@ -25,122 +25,111 @@ doi: XX.XXXXX/phen0000
 
-At the end of this lesson, you will be able to:
+This lesson will demonstrate how to build a digital gazetteer, starting with a simple spreadsheet that you can build into 'Linked Open Data' resources to communicate with other projects. [Linked Data](https://en.wikipedia.org/wiki/Linked_data) is structured data that can be interlinked with other data to undertake large queries. 'Linked Open Data' is Linked Data that is released under an open license, meaning that it can be reused for other projects. You can find out more about Linked Open Data in [this *Programming Historian* introduction to linked data](https://programminghistorian.org/en/lessons/intro-to-linked-data).
 
-- Define what a gazetteer is, understand the concept of place, and distinguish gazetteers from other forms of spatial information. 
-- Identify scenarios for which creating a gazetteer may be preferable to using a geographic information system.  
-- Transform a historical text into a gazetteer.  
-- Share a gazetteer with other platforms to enhance it and use it for analytical purposes. 
+This lesson fulfills two components: first, demonstrating why and how a scholar might choose to build a basic gazetteer and, second, how a gazetteer can support historical analysis. At the end of this lesson, you will be able to:
 
-### Prerequisites
+- Understand the concept of place   
+- Define what a gazetteer is and distinguish it from other forms of spatial information   
+- Identify scenarios for which creating a gazetteer may be preferable to using a geographic information system   
+- Transform a historical text into a gazetteer   
+- Share a gazetteer with other platforms to enhance it and use it for analytical purposes   
 
-No coding experience is needed to complete this lesson.  You should be comfortable designing and using spreadsheets and you will need access to a spreadsheet platform such as Microsoft Excel, Google Sheets, or LibreOffice Calc.
+### Prerequisites
 
-## Historical Example
+No coding experience is needed to complete this lesson. You should simply be comfortable designing and using spreadsheets, and you will need access to a spreadsheet platform such as Microsoft Excel, Google Sheets, or LibreOffice Calc.
 
-This lesson will show you how to create a gazetteer based on the online [Itinerary of Benjamin of Tudela](https://depts.washington.edu/silkroad/texts/tudela.html), an English translation of a Hebrew-language itinerary composed by Benjamin of Tudela (1130-1173), a Jewish traveler who journeyed between the Iberian Peninsula and West Asia in the twelfth century. Benjamin transited through three hundred cities along his route, recording information about geography, ethnography, commerce, Jewish life, and Jewish-Muslim relations.[^2] This text is a major work of medieval geography and Jewish history. You will build a gazetteer of places that Benjamin visited. 
+### Example Case Study
```

Comment:
I've reworded a couple headings. Let me know if you agree with my suggestions!

Reply:
These are good changes. Thanks.